Student manager.js

```diff
@@ -0,0 +1,20 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (studentName === "") {
+        console.log("You cannot add an empty string to a class");
+    } else if (studentName.toLowerCase() === "Queen") {
```

Comment:
The value of the left hand side expression is going to be lowercased, so I guess you meant

```suggestion
    } else if (studentName.toLowerCase() === "queen") {
```